libafl/src/executors/command.rs

```diff
@@ -630,22 +681,16 @@ pub trait CommandConfigurator: Sized + Debug {
         I: Input + HasTargetBytes;
 
     /// Create an `Executor` from this `CommandConfigurator`.
-    fn into_executor<EM, I, OT, S, Z>(self, observers: OT) -> CommandExecutor<EM, I, OT, S, Self, Z>
+    fn into_executor<EM, I, OT, S, Z>(
+        self,
+        observers: OT,
+        has_stdout_observer: Option<String>,
+        has_stderr_observer: Option<String>,
+        has_asan_observer: Option<String>,
+    ) -> CommandExecutor<EM, I, OT, S, Self, Z>
     where
         OT: Debug + MatchName,
     {
-        let has_asan_observer = observers
```

Comment:
@andreafioraldi why didn't you keep it roughly like this, btw? We could match by type and take all of them

Reply:
Because here we were assuming that the names of the observers are fixed. Match by type is a thing in libafl only when using nigthly, specialization is not a stable feature yet.

Reply:
Sad rust noises